packages/loader/container-loader/src/container.ts

```diff
@@ -1429,6 +1477,23 @@ export class Container
 			pendingLocalState?.pendingRuntimeState,
 		);
 
+		// replay saved ops
+		if (pendingLocalState?.savedOps) {
+			for (const message of pendingLocalState.savedOps) {
+				// Process each saved op as if it were a remote message. runtime would not expect
+				// replayed ops to be local, since it did not actually submit them.
+				this.savedOps.push(message);
+				this.protocolHandler.processMessage(message, false);
```

Comment:
if (this.offlineLoadEnabled)? would it be possible to just call this.processRemoteMessage here, rather than these 3 lines, and the emit below? as then the only additional call is  `await this.context.notifyOpReplay(message);`

Reply:
I've done this, please take a look. It requires waiting to set the clientId to the one in the stash blob until after replaying saved ops, which I'm not sure I like better than just explicitly processing them as non-local ops.

The problem is that we need saved ops from the previous clientId to be processed as remote, since runtime has no pending data associated with them, but live ops from the previous clientId need to be processed as local, so runtime can match them to pending stashed ops.